Minor Doc improvements Metal3 (#1011)
suse-edge/suse-edge.github.io: asciidoc/components/metal3.adoc

Metal^3^ provides Kubernetes-native resources to manage the lifecycle of bare-metal servers
which support management via out-of-band protocols such as https://www.dmtf.org/standards/redfish[Redfish].

It also has mature support for https://cluster-api.sigs.k8s.io/[Cluster API (CAPI)] 
which enables management of infrastructure resources across multiple
infrastructure providers via broadly adopted vendor-neutral APIs, and uses

> Metal3 as an infrastructure backend for Machine objects.  

Reviewer: 3 comments here: 
1. We will need to break this into smaller sentences. 
2. Do we need to mention "infrastructure" in all the 3 places, like infrastructure resources, infrastructure providers, and infrastructure backend? 
3. Any specific reason for Capitalizing "Machine"? 
```suggestion
It also has mature support for https://cluster-api.sigs.k8s.io/[Cluster API (CAPI)] . This enables management of infrastructure resources across multiple infrastructure providers via broadly adopted vendor-neutral APIs, and uses Metal^3^ as an infrastructure backend for machine objects.  
```

== How does SUSE Edge use Metal^3^?

SUSE Edge uses Metal^3^ to manage the lifecycle of physical hardware, such as
servers. The hardware needs to support an out-of-band management protocol that

> is supported by Metal3 (e.g. Redfish). When a SUSE Edge management cluster

Author: It should always be Metal^3^. Let me fix

> provisions or deprovisions downstream clusters, Metal^3^ will do things like: +

Reviewer: ```suggestion
provisions or deprovisions downstream clusters, Metal^3^ does the following actions: 
```
